### ISSUE TITLE
[xtremezone] Exception (xtremezone): An established connection was aborted by the software in your host machine: An error occurred while sending the request. (Test)

### ISSUE THREAD

**Issue by scottia** [Jackett/Jackett#2821]
Site is down or link has changed for xtremezone

**Jackett version**:  0.8.815.0
**Mono version** (if not using Windows):  

-------------------------------

**Comment by garfield69** [comment]
yeah. looks like some kind of internal strife has forced a closure. this from https://www.facebook.com/xtremeztracker
```
Salut. Am fost întrebat ce sa întâmplat cu Xtreme Zone. Dragi mei sincer va zic sincer ca cei din staff au pus mana pe sursa muncita de mine peste E ani pe care au furat-o cu o nesimțire ieșita din comun spunând ca AU URCAT ei torente și ca merita site-ul! Site-ul cât și domeniul este munca mea de peste 10 ani. Nopți nedormite, stres, nervi, satisfacție și pasiune. Toate la un loc au făcut ca acest site sa fie ce a fost și ce este. Deci în concluzie am muncit la sursa și mi-a fost furată de membri stafului în care am avut încredere și care mau furat pe fata! Mai mult de 10 % din site nu au modificat ei restul de 90 % e munca mea exclusiva și din cei 10 ani, 7 i-am dedicat exclusiv siteului Xtreme Zone (myxz.org). Domeniu este la mine cât și sursa și torentele cu tot cu conturi. Doar ca nu mai am timpul și răbdarea necesara sa îl pun iar și sa ii dau drumul de unul singur. Problema se poate rezolva foarte ușor dacă mi-se da accesul la sursa înapoi și dacă primesc ce mise cuvine având în vedere ca este site-ul meu domeniul meu! Vor sa ma dea afara... ok sa se ducă sa își cumpere al numele și sa își dacă ei site de la 0 asa cum am făcut ei nu sa fure munca mea și sa își bată efectiv joc de mine și de voi.

In concluzie Numele Xtreme Zone imi apartie si ei l-au furat
domeniu imi apartine si voiau sa il fure si le-a reusit partial
Sursa este munca mea de 10 ani. si nu stiu nici macar niste imagin sa le centreze nu mai zic de multe altele!

Sper ca in seara asta sau maine sa revina Myxz.org intru fel sau altu.

Nu intrati pe alte domenii care auziti. Contin virusi. sau keyloger.

Va multumesc. The King.
```
Google translate did not do a great job, here is what it made of it:
```
Hello. I was asked what happened to Xtreme Zone. My dear, honestly, I will say honestly that the staff have put their hand on the source I worked for over the years that they stole with an outrageous silence saying that they HAVE screamed and deserved the site! The site and field is my work for over 10 years. Sleepless nights, stress, nerves, satisfaction and passion. All together, this site was what it was and what it is. So, in conclusion, I worked at the source and was stolen by members of the staff in whom I trusted and who stole my face! More than 10% of the site did not change the rest of my 90% is my exclusive work and 10 years, 7 I dedicated exclusively to Xtreme Zone (myxz.org). Domain is with me as well as source and torrents with accounts. I just do not have the time and the patience to put it back and let it alone. The problem can be solved very easily if I am given access to the source back and if I receive what my mission is because my site is my domain! They want to get me out ... ok to go buy their name and if their site from 0 as they did they did not steal my work and actually fight against me and you.

In conclusion, Xtreme Zone names belong to me and they stole it
my domain belongs and they wanted to steal it and partially succeeded
The source is my 10-year work. and I do not even know how to focus them, not to mention many more!

I hope that Myxz.org will come back tomorrow or tomorrow in one way or another.

Do not go into other areas that you hear. I have viruses. or keyloger.

Thank you. The King.
```
also
```
Momentan Siteul vostru preferat nu mai merge!
In functie de anumite lucrari care se vor intampla va reveni sau nu!
Va rugam sa asteptati! O sa anunt cand va fi dinou online!

Domeniul ramane in continuare pe http://www.myxz.org si va reveni in curand!
```
```
Your favorite site now does not work anymore!
Depending on some works that will happen will come back or not!
Please wait! I'll let you know when the dino will be online!

The domain remains on http://www.myxz.org and will come back soon!
```

**Comment by dunkf** [comment]
its back up at https://myxz.eu/ getting an error "Login failed: No form found on https://myxz.eu/login using form selector form"

**Comment by garfield69** [comment]
@dunkf  if the login does not work that suggest that address you are using is from one of the competing factions running an alternate site, which is using slightly different html code.

**Comment by dunkf** [comment]
the .eu address has been used for awhile on there tracker but its strange the login page has changed

**Comment by dunkf** [comment]
there site redirection is not working so it needs the .php adding to the links in xtremezone.yml. 
working fine here again but i understand the caution

**Comment by BlackS0ull** [comment]
Hello,

I would like to add again this torrent website but I cannot due the reasons that already written in earlier posts. I wonder if I can do it myself somehow, or I have to ask someone to do it?

Can anyone help me out how I can change the XtremeZone Torrent website address from Myxz.org (which is now configured in Jackett) to the one that is working fine:  https://myxz.eu/ 

Many thanks!

**Comment by scottia** [comment]
https://myxz.eu/ is valid but recommended to use your existing account @  https://www.myxz.eu/login.php to login.

My account wasn't working, couldn't login so I reset the PW and received an email from new admin. with new account details stating to use https://myxz.eu/ in future.

I am logged back in to xtremezone now.  Not sure how Jackett wants to handle since primary admin. claims site ( https://www.myxz.org/ ) was stolen from him.  Am sure more to come based on comments on [FB](https://www.facebook.com/xtremeztracker/).

Here's whats on the new home page @ https://myxz.eu/

Dear users, the XtremeZone site has permanently moved from .org domain to .eu. Some of you may have noticed this address in the last few months in the torrent client. The old .org address is no longer valid. The new site has moved the whole site, you do not need a new account, and you do not have to worry about ration, everything has remained the same. The same torrents, the same staff, the same community, you'll find them all on my new address myxz.eu 

Tell your friends!

**Comment by BlackS0ull** [comment]
Hello Scottia,

Yes, I know those details as I've read on the FB page as well, but sincerely I don't care about their problems. The site is working very nice on the new web address, the one with .eu, so for me is fine :)

No worries, we will wait an update from Jackett :)

**Comment by kaso17** [comment]
Changed to .eu via c11de64a3523deb4a448bc7293d3b45b2840f6fb and 0e38b1ada8e8471ad8fce75b7a8f14578ef15b02
You still have to manually change the site link in the configuration.
Once it's clear that the .org domain won't come back we'll update it automatically (@xtremezone users, please keep us updated about the situation) .

**Comment by scottia** [comment]
Won't allow me to manually change the link.  Attempted to add and change link from .org to .eu

Jackett.Common.ExceptionWithConfigData: Login failed: No form found on https://www.myxz.eu/login using form selector form
   at Jackett.Common.Indexers.CardigannIndexer.<DoLogin>d__15.MoveNext()

**Comment by BlackS0ull** [comment]
Hello,

I have the same exact error that scottia has.

 An error occured while updating this indexer
Login failed: No form found on https://www.myxz.eu/login using form selector form

Thank you in advance!

**Comment by kaso17** [comment]
please update to v0.8.823 and then change the domain

**Comment by BlackS0ull** [comment]
it works perfect now.

many thanks again for your work!

**Comment by BlackS0ull** [comment]
Hello,

It seems that are still some errors when trying to download a torrent from this website :(

NzbDrone.Core.Exceptions.ReleaseDownloadException: Downloading torrent failed ---> NzbDrone.Common.Http.HttpException: HTTP request failed: [404:NotFound] [GET] at [http://127.0.0.1:9117/dl/xtremezone/?jackett_apikey=nqn427y199513zygn1hyodz2hk153ajo&path=dk1yT2dTeWRjNk8ydHY4MXF3U3ZyeTNmUFpUWTNRakwxOUxrSHZQenNaMEgvUXlMQnFyWXE5aFVndzdvWXFsTHdYWkxWNmdNQm9OckpBb09jMStMWGZOUGwwSjI1SzNoY25kbllRUVExMTZqYmpuc2F1aG4xU011M3lJczgway9qTkl5TXByd05KRFlTZ3BMN2pFNEc0NXBtdVkwUHJYWjQ1cG1OYXVLRkNFTzlCVUp3YVY1bDRSVWFobWY0NTIzaDhwSDlRUk9lMGt0ZkV3SzZUTHg4ZS8yQnRQZ2dtQkNRTWc3K2VkZ2xpZmZkNUFETmZGSlRRM1IwR1l6UVdxcGZ6U0dPRjNMRVpVWnUzQUJxMXFQcHJqeEh3SW5RUkE4QlVUVTllT1pnMDFzNDcxdmh2TGdnV0V4cTQ0SXppMTkwUkE3VEVZK3ZDRXUyS3h4Lzl3TU9GYnBWcG9zOVpjTXpXMHBTb3R2cmdjPQ2&file=Coco.2017.1080p.BluRay.x264.RoSubbed-SPARKS]
   at NzbDrone.Common.Http.HttpClient.Execute(HttpRequest request) in C:\projects\radarr-usby1\src\NzbDrone.Common\Http\HttpClient.cs:line 87
   at NzbDrone.Common.Http.HttpClient.Get(HttpRequest request) in C:\projects\radarr-usby1\src\NzbDrone.Common\Http\HttpClient.cs:line 213
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteMovie remoteEpisode, String torrentUrl) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TorrentClientBase.cs:line 222
   --- End of inner exception stack trace ---
   at NzbDrone.Core.Download.TorrentClientBase`1.DownloadFromWebUrl(RemoteMovie remoteEpisode, String torrentUrl) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TorrentClientBase.cs:line 260
   at NzbDrone.Core.Download.TorrentClientBase`1.Download(RemoteMovie remoteMovie) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\TorrentClientBase.cs:line 125
   at NzbDrone.Core.Download.DownloadService.DownloadReport(RemoteMovie remoteMovie, Boolean foceDownload) in C:\projects\radarr-usby1\src\NzbDrone.Core\Download\DownloadService.cs:line 137
   at NzbDrone.Api.Indexers.ReleaseModule.DownloadRelease(ReleaseResource release) in C:\projects\radarr-usby1\src\NzbDrone.Api\Indexers\ReleaseModule.cs:line 73

Unfortunately I can't understand where is exactly the problem as the connection from Jackett is working, also in Radarr is working fine when trying the connection towards this tracker inside the "INDEXERS" tab.

Thank you in advance!

**Comment by kaso17** [comment]
DL link was fixed via 271036b527edbb049df4b36a6677efe031736536
